test/integration/test_utils.mjs

```diff
@@ -461,11 +461,19 @@ async function kbDeleteLastWord(page) {
   }
 }
 
+async function deleteEditors(page) {
+  await page.keyboard.down("Control");
+  await page.keyboard.press("Backspace");
+  await page.keyboard.up("Control");
```

Comment:
I might be missing something, but why can't we just "press" the `Delete` key once instead?